autoscoper/CMakeLists.txt

```diff
@@ -1,171 +1,171 @@
-SET(CMAKE_CURRENT_SOURCE_DIR ${CMAKE_CURRENT_SOURCE_DIR}/src)

```

Comment:
Look like the line endings of text file is inconsistent (dos vs unix), could you check that his is why the complete file ends up being modified ?

Reply:
Yeah that looks like what is happening, I can try updating the script and rebasing the code

Reply:
@jcfr I'm not sure what happening as this only happened in the one files for the [Convert to lower case commit](https://github.com/BrownBiomechanics/Autoscoper/pull/10/commits/b3597627629d33cc3dbed21aaed300a23f49ab15) so I don't think it has to do with the line endings.

Reply:
Thanks for looking into it, I just checked out the branch locally and will follow up in a few minutes.